assets/data/effects/job/job_buildDrauvenOutpost.json

```diff
@@ -99,80 +100,45 @@
 ],
 "outcomes": [
 	{
-		"class": "AddHistory",
-		"target": "npc",
-		"addHistory": [
-			[ "drauvenBase" ]
-		]
+		"class": "ForEach",
```

Comment:
🤔 This logic might want to move to an effect branch so that we can call it from multiple sources. 

For example, we could then run it immediately, but also adds an effect that checks, say, `MONTHLY` for any human farmers that have replaced our Drauven ones and Drauvify any that are found.